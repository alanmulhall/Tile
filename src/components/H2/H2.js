import styled from 'styled-components';

const H2 = styled.h2`
  font-family: ${props => props.theme.fonts.primary};
  font-size: ${props => props.theme.fontScale.m};
  line-height: ${props => props.theme.lineHeight.s};
  margin: 0;
`;

export default H2;
