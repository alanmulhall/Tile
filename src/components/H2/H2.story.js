import React from 'react';
import { storiesOf } from '@storybook/react';
import centered from '@storybook/addon-centered';
import withTests from '../../utils/withTests';
import H2 from './H2';

const H2Story = () =>
  storiesOf('H2', module)
    .addDecorator(centered)
    .addDecorator(withTests('H2'))
    .add('Default', () => (
      <H2>Thatâ€™s been one of my mantras - focus and simplicity.</H2>
    ));

export default H2Story;
