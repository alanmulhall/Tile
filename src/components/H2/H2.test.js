import React from 'react';
import 'jest-styled-components';
import renderer from '../../utils/rendererThemeProviderWrapper.js';
import H2 from './H2';

describe('<H2 />', () => {
  it('renders successfullly', () => {
    const tree = renderer(<H2 />);
    expect(tree).toMatchSnapshot();
  });
});
