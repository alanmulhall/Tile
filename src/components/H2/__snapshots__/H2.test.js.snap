// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<H2 /> renders successfullly 1`] = `
.c0 {
  font-family: Helvetica,Arial,sans-serif;
  font-size: 1rem;
  line-height: 1.2;
  margin: 0;
}

<h2
  className="c0"
/>
`;
